pages/osx/open.md

```diff
@@ -6,9 +6,13 @@
 
 `open {{file.ext}}`
 
-- Run a graphical MacOSX application:
+- Run a graphical macOS application:
 
-`open /Applications/{{Application.app}}`
+`open -a {{Application}}`
+
+- Run a graphical macOS app based on the bundle identifier:
```

Comment:
Can we say something like `Refer to osascript to get the application id` ? And create a separate page for osascript and put this option there ?